codalab/model/bundle_model.py

```diff
@@ -1010,8 +1010,10 @@ def transition_bundle_running(self, bundle, worker_run, row, user_id, worker_id,
             'remote': worker_run.remote,
             'cpu_usage': cpu_usage,
             'memory_usage': memory_usage,
-            'data_size': worker_run.disk_utilization,
         }
+        if self.get_bundle_state(bundle.uuid) != State.FAILED:
```

Comment:
Note: This was another bug that occurred. Basically, in `bundles.py`, the bundle is failed during the `UPLOADING_RESULTS` phase [here](https://github.com/codalab/codalab-worksheets/blob/master/codalab/rest/bundles.py#L1209-L1219) if the user doesn't have enough disk quota to upload the results. When this would happen, the worker isn't updated on the failure status, so it just goes and hits the bundle checkin and finalizes the bundle, and in `transition_bundle_finished` the bundle is marked as `FAILED` because it has a `failure_message` in its metdata from the code in bundles.py. But, when that bundle checkin occurs, the data size is updated again which is WRONG because the actual size of the bundle ON THE SERVER (or Azure blob) is zero even though on the worker it still has a real size (until it gets deleted later)